### DISCUSSION TITLE
brew doctor say Some taps are not on the default git origin branch

### DISCUSSION THREAD

**Discussion by Bruno21** [Homebrew/discussions#2076] [Everyday usage] [answered]
Hello,

I run brew doctor:
```
Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo shivammathur/extensions) checkout master
```

I run git -C $(brew --repo shivammathur/extensions) checkout master

```
git -C $(brew --repo shivammathur/extensions) checkout master
.github/scripts/edit.sh: needs merge
.github/workflows/tests-all.yml: needs merge
Formula/vips@7.0.rb: needs merge
Formula/vips@7.1.rb: needs merge
Formula/vips@7.2.rb: needs merge
Formula/vips@7.3.rb: needs merge
Formula/vips@7.4.rb: needs merge
Formula/vips@8.0.rb: needs merge
Formula/vips@8.1.rb: needs merge
error: vous devez d'abord résoudre votre index courant
```

What should I do ?

**Comment by SMillerDev** [comment, marked as answer]
If you want to keep the changes you made to those formula there is nothing you need to do. `brew doctor` only provides information and if you don't have problems you don't need to change anything.

If you want to discard the changes, the easiest way would be 
```shell
git -C $(brew --repo shivammathur/extensions) add .
git -C $(brew --repo shivammathur/extensions) reset --hard origin/master
git -C $(brew --repo shivammathur/extensions) checkout master
```

**Reply by maiaferrer** [reply]
I am seeing this same display, however I was told it should read "your system is ready for brewing" which is why I am confused

**Comment by Bruno21** [comment]
Thanks for your quick answer SMillerDev.

The last command say:
```
HEAD's previous position was on 13d8b20 imap@8.1: update 8.1.0 bottle.
Switching to the 'master' branch
Your branch and 'origin / master' have diverged,
and have 1 and 54 different commits each respectively.
  (use "git pull" to merge the remote branch into yours)
```

**Reply by SMillerDev** [reply]
If you do `git -C $(brew --repo shivammathur/extensions) reset --hard origin/master` again it should fix this.

**Reply by maiaferrer** [reply]
I received this message after typing that:
[
](fatal: cannot change to '/usr/local/Homebrew/Library/Taps/shivammathur/homebrew-extensions': No such file or directory)

**Reply by carlocab** [reply]
@maiaferrer try
```
rm -rf "$(brew --repo shivammathur/extensions)"
brew tap shivammathur/extensions
```
Note: this will destroy any changes you've made to your local copy of `shivammathur/extensions`.

**Comment by Bruno21** [comment]
Repaired !

Many thanks.
I will backup this thread...
